### PR TITLE
fix: use empty configuration by default

### PR DESCRIPTION
**Pull request by typotter**
Eppo internal
🎟️ ff-3387
related [java-common PR](https://github.com/Eppo-exp/sdk-common-jdk/pull/51)

### CODE REVIEW

File: eppo/build.gradle [hunk 1]
```diff
@@ -7,7 +7,7 @@ plugins {
 }
 
 group = "cloud.eppo"
-version = "4.0.1-SNAPSHOT"
+version = "4.0.2-SNAPSHOT"
```
Comment (typotter) [inline eppo/build.gradle:10]: todo(typotter): bump for next version before merge

File: eppo/build.gradle [hunk 2]
```diff
@@ -68,7 +68,7 @@ ext.versions = [
 ]
 
 dependencies {
-    api 'cloud.eppo:sdk-common-jvm:3.3.2'
+    api 'cloud.eppo:sdk-common-jvm:3.3.4-SNAPSHOT'
```
Comment (typotter) [inline eppo/build.gradle:71]: todo(typotter): use release version before merge

File: eppo/src/main/java/cloud/eppo/android/ConfigurationStore.java [hunk 3]
```diff
@@ -19,14 +19,16 @@ public class ConfigurationStore implements IConfigurationStore {
   private static final String TAG = logTag(ConfigurationStore.class);
   private final ConfigCacheFile cacheFile;
   private final Object cacheLock = new Object();
-  private Configuration configuration;
+
+  // default to an empty config
+  private volatile Configuration configuration = Configuration.emptyConfig();
```
Comment (aarsilv) [inline eppo/src/main/java/cloud/eppo/android/ConfigurationStore.java:24]: 👍

**Review by aarsilv — APPROVED** [review]

File: eppo/src/main/java/cloud/eppo/android/ConfigurationStore.java (hunk 3 above)
Comment (felipecsl) [reply eppo/src/main/java/cloud/eppo/android/ConfigurationStore.java:24]: why volatile? this can be changed from multiple threads?

**Review by felipecsl — APPROVED** [review]
I like the non-null approach 👍🏽

**Pull request merged**